Tests/DataXScenarios/DataXScenarios/InteractiveQueryAndSchemaGenScenarios.cs

```diff
@@ -110,10 +110,10 @@ public static StepResult RefreshSampleAndKernel(ScenarioContext context)
         public static StepResult DeleteKernel(ScenarioContext context)
         {
             var baseAddress = $"{context[Context.ServiceUrl] as string}/api/DataX.Flow/Flow.InteractiveQueryService/kernel/delete";
-            string json = JsonConvert.SerializeObject((string)context[Context.KernelId]);
+            context[Context.DeleteKernelJson] = new Helper().GetDeleteKernelJson(context);
             string jsonResult = Request.Post(baseAddress,
-                    new RequestContent(Encoding.UTF8.GetBytes(json), "application/json"),
-                    context[Context.AuthToken] as string);
+                    new RequestContent(Encoding.UTF8.GetBytes((string)context[Context.DeleteKernelJson]), "application/json"),
+                    bearerToken: context[Context.AuthToken] as string, sslTrust: (bool)context[Context.TrustSsl]);
```

Comment:
can this be refactored similar to above lines?

Reply:
A new simpler method to do GET and POST requests was added